src/redux/getters/getEvents.js

```diff
@@ -53,3 +53,67 @@ export function filterTripToDos(events) {
   });
   return events;
 }
+
+/**
+ * Sorts the events.
+ * @param events array of events, each with a trip_start
+ * and a trip_end time.
```

Comment:
This is missing a @returns tag.

Reply:
It isn't returning anything, just sorting an array in place